egs/wsj/s5/steps/durmod/durmodel_train.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
You can assume that the user is always going to provide all the arguments required to run the script from scratch. This will simplify the script. In case the user wants to avoid the egs preparation stage they can provide a directory where the egs are being stored. 

(see the  --egs-dir option in steps/nnet3/train_tdnn.sh which is used for similar purpose).